lang/compiler.go

```diff
@@ -206,34 +206,49 @@ func buildSpec(t *flux.TableObject, ider flux.IDer, spec *flux.Spec, visited map
 	spec.Operations = append(spec.Operations, t.Operation(ider))
 }
 
-// Program implements the flux.Program interface
-type Program struct {
+// program implements the flux.Program interface
+type program struct {
 	deps     execute.Dependencies
```

Comment:
One thing that I have been dealing with in getting tests to pass in `influxdb` is that the dependencies need to be set so that the executor knows how to execute `from`, `to`, etc.  I filed this issue for it: #1126

In the meantime I have been working around it by making the controller assign the dependencies.  But my workaround won't work if you make `lang.Program` unexported.  Would you mind keeping it public for now, until #1126 is fixed?